packages/core/src/physics/DynamicCollider.ts

```diff
@@ -0,0 +1,54 @@
+import { IDynamicCollider } from "@oasis-engine/design";
+import { Entity } from "../Entity";
+import { Collider } from "./Collider";
+import { PhysicsManager } from "./PhysicsManager";
+import { Vector3 } from "@oasis-engine/math";
+
+/**
+ * A dynamic collider can act with self-defined movement or physical force
+ */
+export class DynamicCollider extends Collider {
+  private readonly _nativeDynamicCollider: IDynamicCollider;
+
+  /** The linear velocity vector of the dynamic collider measured in world unit per second. */
+  linearVelocity: Vector3;
+  /** The angular velocity vector of the dynamic collider measured in radians per second. */
+  angularVelocity: Vector3;
+  /** The linear damping of the dynamic collider. */
+  linearDamping: number;
+  /** The angular damping of the dynamic collider. */
+  angularDamping: number;
+  /** The mass of the dynamic collider. */
+  mass: number;
+  /** Controls whether physics affects the dynamic collider. */
+  isKinematic: boolean;
+
+  /**
+   * The collider attached
+   */
+  get collider(): IDynamicCollider {
+    return this._nativeDynamicCollider;
+  }
```

Comment:
`this._nativeDynamicCollider` don't need to public.

Reply:
done